packages/browser/src/core/queue/event-queue.ts

```diff
@@ -232,9 +232,14 @@ export class EventQueue extends Emitter {
         return true
       }
 
+      const alternativeNameMatch = p.alternativeNames?.filter((name) =>
+        Object.keys(denyList).includes(name)
+      )
+
       // Explicit integration option takes precedence, `All: false` does not apply to Segment.io
       return (
         denyList[p.name] ??
+        alternativeNameMatch ??
```

Comment:
Isn't `alternativeNameMismatch` an array of strings here? I think that'd be evaluated as a truthy value so we'd never hit the next conditional, or am I missing something?

Reply:
Nice catch - fixed the logic and added a new test